anonymizer/OpenDiffix.Service.Tests/Program.Tests.fs

```diff
@@ -5,79 +5,62 @@ open FsUnit.Xunit
 
 open OpenDiffix.Service.Program
 
-let private dataDirectory = __SOURCE_DIRECTORY__ + "/../../data/data.sqlite"
+let private dataPath = __SOURCE_DIRECTORY__ + "/../reference/data/customers.csv"
 
-// [<Fact>]
-// let ``Prints version`` () =
-//   [| "--version" |] |> mainCore |> should not' (be Empty)
+type TempFile() =
+  let path = System.IO.Path.GetTempFileName()
+  member this.Path = path
 
-// [<Fact>]
-// let ``Counts all rows`` () =
-//   [| "-f"; dataDirectory; "--aid-columns"; "customers.id"; "-q"; "SELECT count(*) FROM customers" |]
-//   |> mainCore
-//   |> should not' (be Empty)
+  interface System.IDisposable with
+    member this.Dispose() = System.IO.File.Delete(path)
 
-// [<Fact>]
-// let ``Counts in non-anonymized tables`` () =
-//   [| "-f"; dataDirectory; "-q"; "SELECT count(*) FROM purchases" |]
-//   |> mainCore
-//   |> should not' (be Empty)
+[<Fact>]
+let ``Handles Load request`` () =
+  let request =
+    $"""
+    {{"type":"Load","inputPath":"%s{dataPath}","rows":10000}}
```

Comment:
Why are the double curly brackets needed?

Reply:
this is the way to "escape" `{}` in a `$""""""` literal

Reply:
Probably to print a literal curly instead of interpolation delimiter.

Reply:
Thanks, I was under the impression interpolation happens only after a `%` char.
But it seems the format specifier is optional...